packages/block-editor/src/components/block-list/index.js

```diff
@@ -201,8 +201,8 @@ class BlockList extends Component {
 		} = this.props;
 
 		return (
-			<div className="editor-block-list__layout block-editor-block-list__layout">
-				{ map( blockClientIds, ( clientId ) => {
+			<div className="editor-block-list__layout block-editor-block-list__layout" style={ { position: 'relative' } }>
```

Comment:
Why does this style need to be inline, vs. in the stylesheet?

Reply:
Probably not even necessary, just something forgotten. I'll check.